apps/condo/pages/settings/mobileFeatureConfig/onlyProgressionMeterReadings/index.tsx

```diff
@@ -36,7 +37,7 @@ const OnlyProgressionMeterReadingsContent: React.FC = () => {
                 </Col>
             </Row>
         </>
-    )
+    ) || <Loader fill size='small'/>
```

Comment:
some strange code) It's better to put that loader in a condition before the main return)

Reply:
but it's just a logical expression. if `loading === true` => `Loader` if `loading === false` => content

Reply:
Yes, it's same. But less readable)